test/test_utils_test.go

```diff
@@ -226,8 +227,9 @@ func (ts *ConfigAndClientSuiteBase) InitConfigAndNamespace() error {
 		return err
 	}
 	if ts.config.ShouldRegisterNamespace {
-		err = ts.registerNamespace()
-		if err != nil {
+		if err = ts.registerNamespace(); err != nil {
+			return err
+		} else if err = ts.ensureSearchAttributes(); err != nil {
```

Comment:
For dev server, the `CustomKeywordField` search attribute isn't present like it is in docker compose